data-type/extension/MagiKey-AgeRange.yaml

```
@@ -0,0 +1,25 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: data type
+
+uri: https://armidalesoftware.com/issue/types/AgeRange.htm
+
+specification:
+  - |
+    Age ranges are represented by intervals of years.
+    
+    ```abnf
+    AgeRange = Integer "-" Integer
+    ```
+    
+    This if someone is at least 16 but not yet 26, their age range might be
+    written as `16-26`.
+
+change controller: Armidale Software
+
+used by: [MagiKey]
```

```suggestion
used by: 
  - MagiKey
```

The [format description's style guide](https://gedcom.io/terms/format#readable-yaml) says

> [Sequences](https://yaml.org/spec/1.2.2/#821-block-sequences) and [mappings](https://yaml.org/spec/1.2.2/#822-block-mappings) should be in the block style unless they are empty, in which case a [flow style](https://yaml.org/spec/1.2.2/#74-flow-collection-styles) should be used instead.

This same change would apply in many other places in this PR

---

Done